pkg/reconciler/configuration/configuration.go

```diff
@@ -223,6 +224,7 @@ func CheckNameAvailability(config *v1.Configuration, lister listers.RevisionList
 	} else if !metav1.IsControlledBy(rev, config) {
 		// If the revision isn't controller by this configuration, then
 		// do not use it.
+		logger.Warnf("The revision is not controlled by this configuration. Rev.meta: %#v", rev.GetOwnerReferences())
```

Comment:
does this need to be a Warn or is debug enough for this?

```suggestion
		logger.Debugf("The revision is not controlled by this configuration. OwnerReferences: %#v", rev.GetOwnerReferences())
```

Reply:
yes, just thought it may not easy to detect the situation, and it is currently only seen in our prod env. Then, usually we won't open debug level log on prod env.

Reply:
anyway, I updated to debug now